src/decode.rs

```diff
@@ -4879,6 +4879,18 @@ pub(crate) fn rav1d_decode_frame_exit(
     if c.fc.len() > 1 && retval.is_err() {
         cf.fill_with(Default::default);
     }
+
+    if !retval.is_err() && c.fc.len() > 1 && c.strict_std_compliance {
+        if f.refp.iter().any(|rf| {
+            rf.p.frame_hdr.is_some()
+                && rf.progress.as_ref().unwrap()[1].load(Ordering::Relaxed) == FRAME_ERROR
```

Comment:
```suggestion
                && rf.progress.as_ref().unwrap()[1].load(Ordering::SeqCst) == FRAME_ERROR
```

A C `atomic_load` uses `SeqCst` by default.  We're only using `Relaxed` where C had no atomics and we only use atomics for the borrow checker.